packages/ubuntu_desktop_installer/lib/l10n/app_ca.arb

```diff
@@ -323,8 +323,8 @@
   "@configuringSystem": {},
   "installingSystem": "S’està instal·lant el sistema…",
   "@installingSystem": {},
-  "whoAreYouPageUsernameTooLong": "Aquest nom és massa llarg.",
-  "@whoAreYouPageUsernameTooLong": {},
+  "identityUsernameTooLong": "Aquest nom és massa llarg.",
+  "@identityUsernameTooLong": {},
   "whoAreYouPageShowPassword": "Mostra la contrasenya",
   "@whoAreYouPageShowPassword": {},
```

Comment:
Did you miss that one?

Reply:
:man_facepalming: thanks for catching it!